src/stravalib/model.py

```diff
@@ -946,7 +944,7 @@ class Segment(
     start_longitude: Optional[float] = None
     end_longitude: Optional[float] = None
     starred: Optional[bool] = None
-    pr_time: Optional[int] = None
```

Comment:
@enadeau  i'm just curious - do you think those int assignments were a part of the unexpected breaking behavior? i may have added that missing it was actually an object returned rather than a numeric value

Reply:
No they've always been like that. I just check the blame and this line was not modified when you typed to model file. It's just that now that we publish the type mypy is noticing.